test/functorch/test_eager_transforms.py

```diff
@@ -2078,9 +2078,16 @@ def test_repeated_argnums(self, device, jacapi):
     @jacrev_and_jacfwd
     def test_float_argnums(self, device, jacapi):
         x = torch.randn(3, device=device)
-        with self.assertRaisesRegex(RuntimeError, "must be int or Tuple"):
+        name = jacapi.__name__
+        with self.assertRaisesRegex(
+            TypeError,
+            f"{name}: `argnums` should be int or Tuple[int, ...], got: <class 'float'>",
```

Comment:
Probably being bullied by the regex and how it handles special characters. Maybe try changing it to just "should be int or Tuple"?